lib/struct_test.go

```diff
@@ -0,0 +1,61 @@
+package pitchfork
+
+import (
+	"fmt"
+)
+
+func structmenu_set_dummy(ctx PfCtx, args []string) (err error) {
+	return
+}
+
+func ExampleStructMenu() {
+	/* Create a fake context, this is an example */
+	ctx := testingctx()
+
+	/* The structure we want to convert into a 'set' and 'add' menu */
+	type Example struct {
+		ID           int      `label:"ID" pfset:"nobody" pfget:"user" hint:"The identify of this field"`
+		AField       string   `label:"Field" pfset:"user" pfget:"user" hint:"A Field a user can modify"`
+		AnotherField string   `label:"Another pfset:"sysadmin" pfget:"user" hint:"Another Field that only a sysadmin could modify, but any user can read"`
```

Comment:
go vet: 
lib/struct_test.go:19: struct field tag `label:"Another pfset:"sysadmin" pfget:"user" hint:"Another Field that only a sysadmin could modify, but any user can read"` not compatible with reflect.StructTag.Get: bad syntax for struct tag pair
exit status 1